README.md

```diff
@@ -156,3 +157,22 @@ For example, Juju Dashboard can use the existing controller API to check OpenFGA
 relations. Using that API the dashboard can check the user's relations and
 determine if it should render the `ReBACAdmin` component and associated
 navigation.
+
+## React Query
+
+ReBAC Admin will use the QueryClient from the host application if it exists.
+This allows you to have central control over caching, error handling etc.
+
+**IMPORTANT:** If you've set up React Query you **MUST** set `staleTime` to some
```

Comment:
Just a thought, but wouldn't it be more reliable for us to add a fallback `staleTime` with something like
```
queryClient.setDefaultOptions({
    ...queryClient.getDefaultOptions(),
    queries: {
      ...queryClient.getDefaultOptions().queries,
      staleTime: contextClient?.getDefaultOptions().queries.staleTime ?? 30000,
    },
  });
  ```
instead of relying on the developers to carefully read this part of the docs and set it themselves?

Reply:
I considered this, but there doesn't seem to be a way to limit these options to just the ReBACAdmin requests, so including the ReBACAdmin component will change the behaviour of the host application.

Maybe instead we should throw or log an error if this option isn't set.

Reply:
I think throwing and/or logging an error just to be safe would be a good idea.

Reply:
Done. I think that's much better.